pontoon/base/static/js/translate.js

```diff
@@ -54,16 +63,15 @@ var Pontoon = (function (my) {
         '" data-entry-pk="' + entity.pk + '">' +
         '<span class="status fa' + (self.user.isTranslator ? '' : ' unselectable') + '"></span>' +
         '<p class="string-wrapper">' +
-          '<span class="source-string">' + source_string + '</span>' +
+          '<span class="source-string">' + Pontoon.highlightSearch(source_string, Pontoon.getSearch()) + '</span>' +
           '<span class="translation-string" dir="auto" lang="' + self.locale.code + '">' +
-            self.doNotRender(entity.translation[0].string || '') +
+              Pontoon.highlightSearch(self.doNotRender(entity.translation[0].string || ''), Pontoon.getSearch()) +
```

Comment:
You should replace `Pontoon` with `self`. And - the second parameter (`self.getSearch()`) is always the same, so it does not need to be passed - you should call it in the `highlightSearch()` function.